src/ViewTrait.php

```diff
@@ -189,6 +189,16 @@ public function withLocale(string $locale): static
         return $new;
     }
 
+    /**
+     * Get the specified locale code.
+     *
+     * @return string The locale code.
+     */
+    public function getLocale(): string
```

Comment:
Need to add new method to `ViewInterface` also.

Reply:
After that the PR becomes major.

Reply:
Yes, we should merge this PR before next major version.